src/antares/service/local_services/study_local.py

```diff
@@ -40,3 +40,6 @@ def delete_binding_constraint(self, constraint: BindingConstraint) -> None:
 
     def delete(self, children: bool) -> None:
         raise NotImplementedError
+
+    def create_variant(self, variant_name: str) -> str:
```

Comment:
Should return a Study more than a str
Same remark for the base method and the api implem